docs/deployment.md

```diff
@@ -32,7 +34,12 @@ The steps for tagging a release for the staging server are as follows:
 3. Tag a release locally in git via: `git tag <tag>`
 4. Push the tag to github via: `git push origin --tags`
 
-## Logs
+## How to view the import logs in Google Cloud Platform console
 
-The results of the deploy will be logged to the **#datatools-deploys** channel
-on Slack.
```

Comment:
This is still true and doesn't need to be deleted.

Reply:
I think since @haroldwoo took over we don't log the deploy status to #datatools-deploys anymore, at least from what i've seen so far in that channel. Since we're using jira to communicate with SRE now, the last 2 succesful deploys were not logged to slack, but rather marked DONE in the JIRA ticket.

Reply:
Okay, thanks. That's new to me. In that case, this is good to go. :)